job-server/src/spark.jobserver/JobServer.scala

```diff
@@ -26,6 +26,13 @@ import org.slf4j.LoggerFactory
 object JobServer {
   val logger = LoggerFactory.getLogger(getClass)
 
+  private def setSystemPropertiesFromConfig(config: Config) {
```

Comment:
Yeah, not sure that the code could be located anywhere else....  open to
ideas.

On Sun, Aug 30, 2015 at 11:34 PM, TimMaltGermany <notifications@github.com>
wrote:

> In job-server/src/spark.jobserver/JobServer.scala
> <https://github.com/spark-jobserver/spark-jobserver/pull/232#discussion_r38287266>
> :
>
> > @@ -26,6 +26,13 @@ import org.slf4j.LoggerFactory
> >  object JobServer {
> >    val logger = LoggerFactory.getLogger(getClass)
> >
> > +  private def setSystemPropertiesFromConfig(config: Config) {
>
> @noorul <https://github.com/noorul> Well, this part is easy to test, but
> anyway: where else should we put it? It is essential that this small piece
> of code is executed very early during the start-up process. As we wrote in
> the description for this Pull Request: we would welcome suggestions on how
> to avoid this copying from the config into properties altogether.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spark-jobserver/spark-jobserver/pull/232/files#r38287266>
> .
>



-- 
If you are free, you need to free somebody else.
If you have some power, then your job is to empower somebody else.
   --- Toni Morrison